pow-migration/src/monitor.rs

```diff
@@ -86,11 +124,27 @@ fn is_valid_ready_txn(
     genesis_config_hash: &String,
 ) -> bool {
     // Check if the txn contains extra data and matches our genesis config hash
-    if let Some(txn_data) = &txn.data {
-        if txn_data != genesis_config_hash {
-            return false;
-        }
-    } else {
+    let Some(txn_data) = &txn.data else {
+        return false;
+    };
+
+    if txn_data != genesis_config_hash {
+        return false;
+    }
+
+    // Check for the other readiness criteria
+    block_window.contains(&txn.block_number)
+        && txn.to_address == Address::burn_address().to_user_friendly_address()
+        && txn.value == 1
```

Comment:
Why is this a criteria? It would not be a good idea to pay more than necessary, but it should not impact validity.

Reply:
The same applies for the readiness transactions, we are using the value == 1 as part of the criteria..

should we remove them from both cases?

Reply:
I would say so. Our tool does it with 1 luna, but that is not really any sort of a requirement for validity.